### PR TITLE
Db/add readme

### PR DESCRIPTION
**Pull request by fainashalts**
This PR adds more information to the Truffle DB readme page.

### CODE REVIEW

File: packages/truffle/package.json [hunk 1]
```diff
@@ -58,7 +58,7 @@
     "yargs": "^8.0.2"
   },
   "optionalDependencies": {
-    "@truffle/db": "^0.2.0-0"
+  "@truffle/db": "^0.2.0-0"
```
Comment (fainashalts) [inline packages/truffle/package.json:61]:
```suggestion
      "@truffle/db": "^0.2.0-0"
```

File: packages/db/README.md [hunk 2]
```diff
@@ -1,11 +1,68 @@
...
+- GraphQL
+- Apollo
+- PouchDB
+
+
+## Configuration to enable Truffle DB
+
+Add the following to your `truffle-config.js` file in order to enable Truffle DB
+
+(enabling Truffle DB does not affect Artifacts, but will produce a new `.db`
```
Comment (eggplantzzz) [inline packages/db/README.md:45]: Should "Artifacts" be captialized?

File: packages/db/README.md [hunk 3]
```diff
@@ -1,11 +1,50 @@
 # `@truffle/db`
 
-> TODO: description
+Truffle DB is a complete, flexible system for the querying and mutation
+of Truffle’s smart contract records. Following the data model available [here](https://trufflesuite.github.io/artifact-updates/data-model.html),
+Truffle DB persists smart contract information that is typically found in
+a project's artifacts, and allows for querying this data using `GraphQL`.
+</br>
```
Comment (fainashalts) [inline packages/db/README.md:7]:
```suggestion
</br>

```

Comment (fainashalts) [inline packages/db/README.md:7]:
```suggestion

</br>
```

File: packages/db/README.md [hunk 4]
```diff
@@ -1,11 +1,51 @@
...
+
+Truffle artifacts are currently stored as JSON files in a project’s `build/contracts/`
+directory. These documents are currently the canonical source of information about
+a project’s smart contracts. While useful, artifacts can be limited for certain advanced
+use cases.
+</br>
```
Comment (fainashalts) [inline packages/db/README.md:14]:
```suggestion

</br>

``

File: packages/db/README.md (hunk 2 above)
Comment (fainashalts) [reply packages/db/README.md]: Fixed! Thanks for pointing this out!

**Review by eggplantzzz — APPROVED** [review]
nice

**Pull request merged**